libr/core/cmd_print.c

```diff
@@ -1629,6 +1629,7 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 	free (string2);
 	free (string);
 	free (s);
+	free (str);
```

Comment:
well, coverity complains but maybe you are right and it is not always allocated.
Maybe we can free after "flag = r_flag_get_i2 (core->flags, ptr);". Seems better to you?